tests/unit/test_client.py

```diff
@@ -1,89 +1,129 @@
-# from typing import Any, Iterator
-#
-# from hiro_graph_client import PasswordAuthTokenApiHandler, AbstractTokenApiHandler, HiroGraph
-# from hiro_graph_client.batchclient import HiroResultCallback, HiroGraphBatch
-#
-#
-# class RunBatch(HiroResultCallback):
-#     hiro_batch_client: HiroGraphBatch
-#
-#     def __init__(self, api_handler: AbstractTokenApiHandler):
-#         self.hiro_batch_client = HiroGraphBatch(
-#             callback=self,
-#             api_handler=api_handler)
-#
-#     def result(self, data: Any, code: int) -> None:
-#         print('Data: ' + str(data))
-#         print('Code: ' + str(code))
-#
-#     def _run(self, commands: Iterator[dict]):
-#         self.hiro_batch_client.multi_command(commands)
-#
-#
+from typing import Any, Iterator
+
+from hiro_graph_client import PasswordAuthTokenApiHandler, AbstractTokenApiHandler, HiroGraph, SSLConfig
+from hiro_graph_client.batchclient import HiroResultCallback, HiroGraphBatch
+
+
+class RunBatch(HiroResultCallback):
+    hiro_batch_client: HiroGraphBatch
+
+    def __init__(self, api_handler: AbstractTokenApiHandler):
+        self.hiro_batch_client = HiroGraphBatch(
+            callback=self,
+            api_handler=api_handler)
+
+    def result(self, data: Any, code: int) -> None:
+        print('Data: ' + str(data))
+        print('Code: ' + str(code))
+
+    def run(self, commands: Iterator[dict]):
+        self.hiro_batch_client.multi_command(commands)
+
+
 class TestClient:
-    # hiro_api_handler = PasswordAuthTokenApiHandler(
-    #     root_url='',
-    #     username='',
-    #     password='',
-    #     client_id='',
-    #     client_secret='',
-    #     secure_logging=False,
-    #     custom_endpoints={
-    #         'graph': '/api/graph/7.2'
-    #     },
-    #     headers={
-    #         'user-agent': 'TestClient'
-    #     }
-    # )
-    #
+    hiro_api_handler = PasswordAuthTokenApiHandler(
+        root_url="https://ec2-3-250-135-44.eu-west-1.compute.amazonaws.com:8443",
+        username='haas1000-connector-core',
+        password='j9dad7gond4ls2taol37ulk56%1aZ',
+        client_id='cju16o7cf0000mz77pbwbhl3q_ckqjkfc0q08r90883i7x521sy',
+        client_secret='978fa4385da282ed8190b12e9ac70ed6e65ea750f6b5282c0205a9d049913ae9f2841998c2ef13bb14db2d6cee0fd1ca9834563865b8f45c555e6ad3dd1be36a',
+        # root_url="https://core.arago.co",
+        # username='haas00019-service-dentsu-connector-dv360v2',
+        # password='pAC45246B-4A81-435F-BB92-EB35F2623036',
+        # username="haas00019-service-dentsu-haas-connector-tradedesk",
+        # password="rBqGMgclgIMN%1aZ",
```

Comment:
Dude 😂

Reply:
Will need to squash to remove from history too

Reply:
🤦 These should NOT have been published. Dammit.